taqueria-plugin-taquito/originate.ts

```diff
@@ -184,9 +186,9 @@ const createBatch = async (parsedArgs: Opts, tezos: TezosToolkit, destination: s
 				}
 			} else {
 				sendErr(
-					msg
-						+ " - There was a problem communicating with the chain. Check the RPC URL of the network or sandbox you're targeting in config.json.\n",
+					"There was a problem communicating with the chain. Perhaps review your RPC URL of the network or sandbox you're targeting.\n",
```

Comment:
Let's avoid the use of weak/indirect language like 'perhaps'

Also, the e2e tests need to be updated as they fail when trying to match this output string

Reply:
Hmm, odd - this looks like another merge issue. That makes me nervous. I didn't purposely revert the messaging (that I added in a previous PR)

Reply:
I've restored the original msging and adjusted the last msg to be stronger.